dbms/src/Storages/DeltaMerge/Segment.cpp

```diff
@@ -679,7 +679,7 @@ SegmentPtr Segment::applyMergeDelta(DMContext & context,
                                     WriteBatches & wbs,
                                     const StableValueSpacePtr & new_stable) const
 {
-    LOG_FMT_INFO(log, "Before apply merge delta: {}", info());
+    LOG_FMT_INFO(log, "Begin segment applyMergeDelta, segment={}", simpleInfo());
```

Comment:
I think using `info()` in the beginning of `applyMergeDelta` is better. So that we can know what's the difference before and after apply merge delta

Reply:
Oh, I see. You put it into one logging line with `old_me` and `new_me` and just log a simpleInfo here.

Reply:
Yes, the full info is usually placed in the log lines when function exits.